include/wil/common.h

```diff
@@ -457,13 +457,13 @@ WI_HEADER_INITITALIZATION_FUNCTION(InitializeDesktopFamilyApis, []
 The above example is used within WIL to decide whether or not the library containing WIL is allowed to use
 desktop APIs.  Building this functionality as #IFDEFs within functions would create ODR violations, whereas
 doing it with global function pointers and header initialization allows a runtime determination. */
-#define WI_HEADER_INITITALIZATION_FUNCTION(name, fn)

```

Comment:
Unfortunately we have 117+ instances of the misspelling in the Windows tree, so we'll need to global search-and-replace when this merges back in, @dunhor

Reply:
Yeah, we probably need to keep misspellings in actual names (functions, macros, etc.). We can define both and have the old misspelled one "forward" to the new correct spelling

Reply:
Argh, I can't make a suggestion outside of this `#if` chain because GitHub doesn't let me add comments to lines unchanged... This can be solved by the following:
```c++
#define WI_HEADER_INITITALIZATION_FUNCTION(name, fn) WI_HEADER_INITIALIZATION_FUNCTION(name, fn)
```

ALTERNATIVELY if this isn't likely used all that much in the wild, we can throw this definition in the "internal" header so that folks in the OS repo don't get broken

Reply:
@dunhor rebased and added the old spelling per your suggestion.